lib/puppet/datatypes/error.rb

```diff
@@ -5,7 +5,7 @@
       issue_code => Optional[Variant[String,Regexp,Type[Enum],Type[Pattern],Type[NotUndef],Type[Undef]]]
     },
     attributes => {
-      message => String[1],
+      msg => String[1],
```

Comment:
I think we need to have the "message" accessible at `.message` I don't care strongly whether that is instead of or in addition to `.msg` though I slightly prefer the latter.

Reply:
You mentioned earlier including `message` as an accessor, but I don't see it.